build/chip/tools.gni

```diff
@@ -24,4 +24,6 @@ declare_args() {
        (current_os != "android" && current_os != "freertos" &&
         current_os != "zephyr" && current_os != "mbed" &&
         current_os != "webos"))
+
+  chip_can_build_all_clusters_app = false
```

Comment:
need some documentation here - what is this flag and when would it be set.

Why is "can build all-clusters app" something that chip_tools would be concerned with?

Reply:
Acked, moved to BUILD.gn